SwiftIsland/Utilities/DataManager.swift

```diff
@@ -8,10 +8,32 @@
 
 import Foundation
 
+protocol Listable: Codable {}
+enum ListableType {
+  case mentor, area
+
+  var endpoint: APIManager.Endpoint {
+    switch self {
+    case .mentor:
+      return APIManager.Endpoint.mentors
```

Comment:
There is no need for the `APIManager.Endpoint` definition here, since `endpoint` already sets its type decleration to `APIManager.Endpoint`.